quick_start/requirements.txt

```diff
@@ -0,0 +1,3 @@
+azure_storage==0.37.0
```

Comment:
You need to have "azure.kusto.data" and "azure.kusto.ingest" in the requirements, and remove storage